spec/models/contact_email_spec.rb

```diff
@@ -14,4 +14,18 @@
   it "belongs to a work" do
     expect(contact_email.emailable).to eq work
   end
+
+  describe "whitespace" do
+    let(:contact_email) do
+      create(
+        :contact_email,
+        email: " timbl@w3.org ",
+        emailable: work
+      )
+    end
+
+    it("strips email") do
```

Comment:
I think you can drop the parens here but NBD

Reply:
I switched it to use no parens on one line and now I get a rubocop warning:

```
spec/models/contact_email_spec.rb:20:7: C: [Correctable] FactoryBot/ConsistentParenthesesStyle: Prefer method call with parentheses
```